### PR TITLE
Fix search path update

### LINKED ISSUE #282: Search bar text is not updated to browsed folder path
ui_search_bar

### PR DESCRIPTION
**Pull request by integricho**
-Fix current path display in search input
-Fix ajax loading of folders
-Fix ajax loading of go up / back button

This fixes #282

### CODE REVIEW

File: librarian/views/filemanager/_main.tpl [hunk 1]
```diff
@@ -48,15 +48,20 @@ def get_views(facet_types):
         ## - Link to complete file list if there is search query
         ## - Link to complete file list if invalid path is requested
         ## - Link to parent directory if no search query and not at top-level
+        <% h.urlquote %>
```
Comment (foxbunny) [inline librarian/views/filemanager/_main.tpl]: Is this one of those peek-a-boo issues where it doesn't work unless you do this?

Comment (integricho) [inline librarian/views/filemanager/_main.tpl]: Yes,exactly, not found

Comment (integricho) [inline librarian/views/filemanager/_main.tpl]: missing comment

**Comment by foxbunny** [comment]
This should be cherry-picked into a hotfix branch.

**Pull request merged**